### PR TITLE
N5  layout corrections

### PR DESCRIPTION
**Pull request by ikruglik**

### CODE REVIEW

File: src/app/ui/admin/issue-details/issue-details.component.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
                 <input type="radio" name="options" autocomplete="off">Resolve
             </label>
         </div>
-        <button class="btn btn-green ml-5" type="button" (click)="patchIssue()">Save</button>
+        <button class="btn btn-green" type="button" (click)="patchIssue()">Save</button>
```
Comment (lukapr) [inline src/app/ui/admin/issue-details/issue-details.component.html]: Could you please move the "Save" button on the "Delete" button place (under the statuses button) and move the "Delete" button on the same line as statuses buttons but on the right side of the page?
Like this:
image

File: src/app/ui/new-issue/issue/issue.component.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <app-header></app-header>
 <app-header-secondary [title]="headerTitle"></app-header-secondary>
-<app-loader *ngIf="!item"></app-loader>
+<app-loader *ngIf="loading"></app-loader>
+<div *ngIf="!loading && !item" style="font-size: 2em; font-weight: bold; width: 300px; margin: auto;">Item not found</div>
```
[the remark this answers is not in the document]
Comment (ikruglik) [reply src/app/ui/new-issue/issue/issue.component.html]: Right, did not notice that was already implemented as I had created my branch from master, not integration.

File: src/app/ui/admin/issue-details/issue-details.component.html (hunk 1 above)
Comment (ikruglik) [reply src/app/ui/admin/issue-details/issue-details.component.html]: Fixed, thank you.

File: src/app/ui/admin/issue-details/issue-details.component.html [hunk 3]
```diff
@@ -1,8 +1,7 @@
 <app-header></app-header>
 <nav class="navbar navbar-light" style="background-color: #a6baa9">
     <a class="navbar-brand navbar-brand-secondary">
-        <button class="btn btn-sm btn-green mr-1" type="button" [routerLink]="['/admin']"><
-        </button>
+        <button class="btn btn-sm btn-green mr-1" type="button" [routerLink]="['/admin']"></button>
         Admin view
```
Comment (lukapr) [inline src/app/ui/admin/issue-details/issue-details.component.html]: There should be `<` text in the button. 
Like this:
image

Could you please revert this changes?

File: src/app/ui/admin/issue-details/issue-details.component.html [hunk 4]
```diff
@@ -42,8 +41,8 @@
                 <input type="radio" name="options" autocomplete="off">Resolve
             </label>
         </div>
-        <button class="btn btn-green ml-5" type="button" (click)="patchIssue()">Save</button>
         <div>
+            <button class="btn btn-green" type="button" (click)="patchIssue()">Save</button>
```
Comment (lukapr) [inline src/app/ui/admin/issue-details/issue-details.component.html]: Buttons now looks very strange:
image

Could you please move the Delete button on the same line as Status buttons (button group) but move it to the right of the screen? I've attached the screenshot on how we want this to be in prev comments.

File: src/app/ui/new-issue/issue/issue.component.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (lukapr) [reply src/app/ui/new-issue/issue/issue.component.html]: So can we remove this code (regarding item not found) from here? It is not used now.

[the remark this answers is not in the document]
Comment (ikruglik) [reply src/app/ui/new-issue/issue/issue.component.html]: Removed.

File: src/app/ui/admin/issue-details/issue-details.component.html (hunk 3 above)
Comment (ikruglik) [reply src/app/ui/admin/issue-details/issue-details.component.html]: Done.

File: src/app/ui/admin/issue-details/issue-details.component.html (hunk 4 above)
Comment (ikruglik) [reply src/app/ui/admin/issue-details/issue-details.component.html]: Fixed, thank you.

File: src/app/ui/admin/issue-details/issue-details.component.html [hunk 5]
```diff
@@ -39,12 +38,14 @@
             </label>
             <label class="btn btn-secondary btn-status"
                    [ngClass]="{'btn-green': status === 'Resolved'}" id="resolve" (click)="chooseStatus('Resolved')">
-                <input type="radio" name="options" autocomplete="off">Resolve
+                <input type="radio" name="options" autocomplete="off">Resolveee
             </label>
```
Comment (lukapr) [inline src/app/ui/admin/issue-details/issue-details.component.html]: "Resolveee"?

Comment (ikruglik) [reply src/app/ui/admin/issue-details/issue-details.component.html]: Oops, sorry

Comment (ikruglik) [reply src/app/ui/admin/issue-details/issue-details.component.html]: Fixed.
